advisories/github-reviewed/2024/03/GHSA-cxjh-pqwp-8mfp/GHSA-cxjh-pqwp-8mfp.json

```diff
@@ -66,10 +66,6 @@
     {
       "type": "PACKAGE",
       "url": "https://github.com/follow-redirects/follow-redirects"
-    },
-    {
-      "type": "WEB",
-      "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/VOIF4EPQUCKDBEVTGRQDZ3CGTYQHPO7Z"
     }
```

Comment:
Didn't remove this explicitly, I think the UI did it? I didn't modify that section.